python/main.py

```diff
@@ -64,23 +67,36 @@ def hello():
 class AddItemResponse(BaseModel):
     message: str
 
+class GetItemResponse(BaseModel):
+    items: List
+
 
 # add_item is a handler to add a new item for POST /items .
 @app.post("/items", response_model=AddItemResponse)
 def add_item(
```

Comment:
```suggestion
async def add_item(
```
Line 113, the function itself is async so the endpoint needs to be async as well.
関数自体がAsyncになっておりますのでEndpointもAsyncにする必要があります。
とは言え、こちらのTaskではAsyncが無くても動けると思います。